src/Symfony/Component/Serializer/Normalizer/ChainNormalizerInterface.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Serializer\Normalizer;
+
+/**
+ * This holds a collection of Normalizers.
+ *
+ * @author Tobias Nyholm <tobias.nyholm@gmail.com>
+ */
+interface ChainNormalizerInterface extends NormalizerInterface
+{
+    /**
+     * Add a Normalizer last in the line.
+     */
+    public function addNormalizer(NormalizerInterface $normalizer): void;
+
+    /**
+     * Add a Normalizer first in line.
+     */
+    public function prependNormalizer(NormalizerInterface $normalizer): void;
```

Comment:
I don't get the purpose of that method

Reply:
They were for altering the service at runtime. They are never used and I should remove them.

Thank you